<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pk</class>
 <widget class="QWidget" name="Pk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>801</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>出招</string>
    </attribute>
    <widget class="QPushButton" name="skill1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>技能1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skill2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>技能2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skill3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>技能3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skill4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>技能4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>换宠</string>
    </attribute>
    <widget class="QListWidget" name="poks">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>671</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>换宠</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>541</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="pet" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>80</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="poke_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="attr2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="attr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="pet_2" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="poke_img_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="attr2_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="attr_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_hp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hp_2">
   <property name="geometry">
    <rect>
     <x>961</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>第</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="huihe">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>回合  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="hp_bar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="hp_2_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
